front/src/styles/scss/applications/operationalStudies/_managetrainschedule.scss

```diff
@@ -42,6 +42,10 @@
   }
 }
 
+.gap-10 {
```

Comment:
Too generic name to be placed on a specific component scss file (place it inside common for example), or rename/nest it inside scss tree linked to the component.

Reply:
done